tests/integration/records/serializers/json/test_authors.py

```diff
@@ -0,0 +1,340 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import json
+
+from helpers.providers.faker import faker
+from invenio_accounts.testutils import login_user_via_session
+
+from inspirehep.accounts.roles import Roles
+
+
+def test_literature_application_json_authors(api_client, db, create_record):
```

Comment:
The test names are inconsistent, we can rename them to something like:

* ``test_literature_authors``
* ``test_literature_authors_detail``
* ``test_literature_authors_detail_with_login...``
* ``test_literature_authors_search_with_login...``